### PR TITLE
Feature#34 profile design

### LINKED ISSUE #34: Profile Design
# REQUIREMENTS
- Similar design to the mockup

# DESIGN
image

# Acceptance Criteria
List all required acceptance criteria
- [ ] Still works
- [ ] Similar to mockup

### PR DESCRIPTION
**Pull request by DerekAyala**
# Description
What did you implement/Why did you implement this:
 - Add a design on the show profile view base if the user doesn't have a skill or a social network.
 - change the container to cont to avoid bootstrap overwritting.
 - fix background color on the application
 - add a back button.
 - Logos to Facebook, Instagram, Twitter, and LinkedIn.
 - SVG images to back, new, edit and delete actions.

  ## Screenshots 
if you don't have a skill
Screenshot from 2022-02-17 18-04-42
if you don't have a social network added
Screenshot from 2022-02-17 18-08-18
profile design
Screenshot from 2022-02-17 17-46-31

### CODE REVIEW

File: app/assets/stylesheets/application.scss [hunk 1]
```diff
@@ -24,12 +27,18 @@ html {
...
+  max-width: 120rem;
   width: 95%;
 }
 
-.main{
+.back {
+  height: 3rem;
+  margin-bottom: 1rem;
+}
+
```
Comment (diegoOG09) [inline app/assets/stylesheets/application.scss:40]: Try not to leave so many spaces

**Review by diegoOG09 — APPROVED** [review]
LGTM

**Review by IrenehMDev — APPROVED** [review]
LGTM

**Review by JuanequeX — APPROVED** [review]
LGTM

**Pull request merged**